domain-models-runtime/src/main/java/org/folio/rest/persist/Criteria/Criteria.java

```diff
@@ -114,15 +114,15 @@ private void createFromSnippet() {
       from = new From();
       from.setSnippet(ARRAY_FROM_CLAUSE + "("
         + PostgresClient.DEFAULT_JSONB_FIELD_NAME + GET_JSON_FIELD + arrayField + ")");
-      from.setAsValue(field.get(0).replaceAll("^'|'$", "")); //remove ''
+      from.setAsValue(field.get(0).replace("'", "")); //remove ''
```

Comment:
The old code removes leading and tailing single quotes: `'Rock'n'Roll'` results in `Rock'n'Roll`

The new code removes all single quotes: `'Rock'n'Roll'` results in `RocknRoll`

I would introduce a new method for the task:

```
  private static String unwrapFromSingleQuotes(String s) {
    if (s.length() >= 2 && s.startsWith("'") && s.endsWith("'")) {
      return s.substring(1, s.length() - 1);
    }
    return s;
  }
```